kitchen.yml

```diff
@@ -58,3 +58,8 @@ suites:
       - centos-8
       - amazonlinux-2
       - fedora-31
+  - name: invalid-conf
+    run_list:
+      - recipe[test::invalid-conf]
+    includes:  # testing invalid config logic works, no need to run on all platforms
```

Comment:
Any reason why we shouldn't though? You never known that it might not work on some platforms and it's always good to check I think. For example, different platforms might have different versions installed and the flag may work differently for whatever reason.

Reply:
sure, was just saving testing cycles but im not opposed to it. ill enable it everywhere